Sources/Build/BuildPlan.swift

```diff
@@ -2112,8 +2208,8 @@ public class BuildPlan {
 
         // add test manifest targets
         if case .manifest = buildParameters.testDiscoveryStrategy {
-            if product.type == .test, let testManifestTarget = testManifestTargetsMap[product] {
-                staticTargets.append(testManifestTarget)
+            if product.type == .test, let testManifestTarget = derivedTestTargetsMap[product] {
+                staticTargets.append(contentsOf: testManifestTarget)
```

Comment:
is testManifestTarget an array?

Reply:
Yeah - I needed to change the type of this dictionary property to make its value an array of targets, since there may now be two synthesized targets. I'll rename this variable on the line you highlighted to be plural, to clarify that.